Common/src/Utils/LoggerInit.cs

```diff
@@ -38,7 +38,7 @@ public LoggerInit(IConfiguration configuration)
                                                     .Enrich.FromLogContext()
                                                     .Enrich.WithProperty("CoreVersion",
                                                                          typeof(LoggerInit).Assembly.GetName()
-                                                                                           .Version)
+                                                                                           .Version!)
```

Comment:
Are we guaranteed it is non null?
```suggestion
                                                                                           .Version ?? "Unknown")
```